content/posts/numpy/mukulikapahari/index.md

```diff
@@ -3,7 +3,7 @@ title: "NumPy Contributor Spotlight: Mukulika Pahari"
 date: 2022-04-12
 draft: false
 description: "An interview with Mukulika Pahari, a NumPy contributor."
-categories: ["contributor spotlight", "numpy"]
+tags: ["contributor spotlight", "numpy"]
```

Comment:
I noticed this elsewhere, apparently tags cannot have spaces. So I used `-` to separate words.